src/Vocup/Properties/Words.de.resx

```diff
@@ -202,6 +202,9 @@
   <data name="VocupVocabularyBookFile" xml:space="preserve">
     <value>Vocup Vokabelheft-Datei</value>
   </data>
+  <data name="VocupVocabularyBookLegacy" xml:space="preserve">
+    <value>Vocup Vokabelheft (veraltet)</value>
```

Comment:
I'm not sure if "veraltet" is clear enough. It could also be interpreted as "alt/datiert" which might be confusing if the book was recently used while practicing. Not sure though how this is in German language. Perhaps name it just (file version 1.0) or something? Need to think this one over... Also in Dutch I would suggest using just the word (legacy) if you don't want to rename this.